bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/BugsnagExitInfoPlugin.kt

```diff
@@ -0,0 +1,53 @@
+package com.bugsnag.android
+
+import android.app.ActivityManager
+import android.content.Context
+import android.os.Build
+import androidx.annotation.RequiresApi
+
+@RequiresApi(Build.VERSION_CODES.R)
+class BugsnagExitInfoPlugin(
```

Comment:
To keep this easy to use from Java, we should add `@JvmOverloads` to the constructor
```suggestion
class BugsnagExitInfoPlugin @JvmOverloads constructor(
```